operations/pyroscope/helm/pyroscope/values-micro-services.yaml

```diff
@@ -84,5 +84,27 @@ pyroscope:
         # Depending on this flag and the number of tenants + blocks that need to be synced on startup, pods can take
         # some time to become ready. This value can be used to ensure Kubernetes waits long enough and reduce errors.
         initialDelaySeconds: 60
+    tenant-settings:
+      kind: StatefulSet
+      replicaCount: 1
+      persistence:
+        enabled: false
+      resources:
+        limits:
+          memory: 4Gi
+        requests:
+          memory: 16Mi
+          cpu: 0.1
+    ad-hoc-profiles:
+      kind: StatefulSet
```

Comment:
should these be deployments instead?

Reply:
Yes you are right, 9b0e996f612bd939917c6d896058efe5597b9a8c